examples/05_glm_second_level/plot_second_level_two_sample_test.py

```diff
@@ -63,18 +63,20 @@
 # Next, we model the effect of conditions (sample 1 vs sample 2).
 import numpy as np
 
-condition_effect = np.hstack(([1] * n_subjects, [-1] * n_subjects))
+condition_effect = np.hstack(([1] * n_subjects, [0] * n_subjects))
 
 # %%
-# The design matrix for the unpaired test doesn't need any more columns
+# The design matrix for the unpaired test needs to add an intercept,
 # For the paired test, we include an intercept for each subject.
 subject_effect = np.vstack((np.eye(n_subjects), np.eye(n_subjects)))
 subjects = [f"S{i:02d}" for i in range(1, n_subjects + 1)]
 
 # %%
 # We then assemble those into design matrices
-unpaired_design_matrix = pd.DataFrame(
-    condition_effect[:, np.newaxis], columns=["vertical vs horizontal"]
+unpaired_design_matrix = pd.DataFrame({
```

Comment:
Black is complaining about the formatting here:
https://github.com/YCHuang0610/nilearn/actions/runs/8893790642/job/24420709786#step:5:99

This should be fixed by applying all our style checks and autofixes and committing the changes.

```bash
pip install tox
tox 
```